src/utils.c

```diff
@@ -170,7 +170,8 @@ ts_interval_value_to_internal(Datum time_val, Oid type_oid)
 			if (interval->month != 0)
 				ereport(ERROR,
 						(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-						 errmsg("intervals must be defined in terms of days or smaller")));
+						 errmsg("months and years not supported"),
+						 errdetail("an interval must be defined as a fixed duration (such as weeks, days, hours, minutes, seconds, etc.)")));
```

Comment:
```suggestion
						 errdetail("An interval must be defined as a fixed duration (such as weeks, days, hours, minutes, seconds, etc.)")));
```
Nit: In PG error style detail messages should start capitalized and end in a period. Main message should start with lowercase and have no period.